lib/curator.ex

```diff
@@ -71,6 +82,32 @@ defmodule Curator do
     # end)
   end
 
+  @doc """
+  Call an extension on all modules
+
+  This provides a way to coordinate actions between the modules, without them knowing about eachother directly.
+  Each module can have verious extensions it broadcasts. The other modules decide if they'll listen...
+  """
+  # def extension(mod, fun, args) do
```

Comment:
@aditya7iyengar Thoughts on this? I was originally going to have it call an 'extension' method that all extensions would have to implement (there's commented out code to accomplish that). I didn't like the arity issue though (and you couldn't override and fallback to a default). I decided instead to call the method directly (if it's exported). Sound good?

Reply:
Decided to call the method directly